src/Tag.js

```diff
@@ -83,19 +83,19 @@ class Tag {
     }
 
     write(stream, vrType, values, syntax, writeOptions) {
-        var vr = ValueRepresentation.createByTypeString(vrType),
-            useSyntax = DicomMessage._normalizeSyntax(syntax);
-
-        var implicit = useSyntax == IMPLICIT_LITTLE_ENDIAN ? true : false,
-            isLittleEndian =
-                useSyntax == IMPLICIT_LITTLE_ENDIAN ||
-                useSyntax == EXPLICIT_LITTLE_ENDIAN
-                    ? true
-                    : false,
-            isEncapsulated =
-                this.isPixelDataTag() && DicomMessage.isEncapsulated(syntax);
-
-        var oldEndian = stream.isLittleEndian;
+        const vr = ValueRepresentation.createByTypeString(vrType);
```

Comment:
Just some cleanup to this method to use the newer syntax and declare 1 value per declaration rather than using , to declare multiples.